docs/contributing.md

```diff
@@ -3,327 +3,376 @@ title: Contributor's guide
 slug: /contributing
 ---
 
-# Contributing
+# Contributing to Bluefin Documentation
```

Comment:
This section should be about contributing to bluefin itself, not the documentation. Add the documentation section in it's own section at the end of the document instead.